your-character/attributes.adoc

```diff
@@ -23,31 +23,103 @@ For example, if your character has 17 strength, it will get 4 attribute modifier
 
 == Strength
 
-Describes the raw power and endurance of the character.
+The *Strength* attribute describes the sheer power and endurance of the character.
+
+.Properties affected by Strength
+* Meele damage
+* Meele attack
 
 === Stamina
 
-Stamina describes the character's muscle efficency. How long can it lift weights without getting tired.
+The *Stamina* attribute describes the character's muscle efficiency. For example, how long can it lift weights or do exhausting activities without getting tired.
+
+.Properties affected by Stamina
+* Carrying capacity
+* Fatigue & Exhaustion
+* Springing
+* Proficiencies
 
 === Muscle
 
-...
+The *Muscle* attribute measures the character's sheer power. A character with high Muscle looks very muscular and able to lift or bend items easily.
+
+.Properties affected by Muscle
```

Comment:
I think carrying capacity should be affected by Muscle as well as Stamina.

Reply:
You are probably right about this one! We need to come up with a formula that considers both.